### ISSUE TITLE
Problem accessing public properties of an extended class

### ISSUE THREAD

**Issue by pabloperich** [jstachio/jstachio#397]
I am using JStach.io to fill a big template with the data of an object with many properties, so I decieded to use public properties instead of getters for simplicity, but I am having problems accessing public properties of an extended class. Theese are my example classes:

```java
@JStache(
        template = """
                {{fieldA}}
                {{fieldB}}
                {{modelC.fieldC}}
                """
)
public class ModelA extends ModelB {
    public String fieldA;
}

public class ModelB {
    public String fieldB;
    public ModelC modelC;
}

public class ModelC {
    public String fieldC;
}
```
If i try to compile this code I get this compilation error:
```
ModelA#template:2: error: Variable not found.
            var: 'fieldB'
       position: 'com.axteroid.prueba.ModelA#template[2:11]'
       template: com.axteroid.prueba.ModelA#template
         reason: Field not found in current context: 'fieldB' , template: com.axteroid.prueba.ModelA#template
  context stack: 
  	<- DeclaredTypeRenderingContext [
  		expression=JavaExpression [text=data, type=com.axteroid.prueba.ModelA, path=[]],
  		definitionElement=com.axteroid.prueba.ModelA,
  		parent=io.jstach.apt.internal.context.RootRenderingContext@7ab1127c]
  	<- io.jstach.apt.internal.context.RootRenderingContext
  {{fieldB}}
            ^
  symbol: mustache directive
  location: mustache template
```

but just modifying the ModelB class to (adding getters):
```java
public class ModelB {

    public String fieldB;
    public ModelC modelC;

    public String getFieldB() {
        return fieldB;
    }

    public ModelC getModelC() {
        return modelC;
    }
}
```

I can copile them successfully. So the problem is only to access the properties of the extended class.

Is there an issue with extended classes?

**Comment by agentgt** [comment]
> Is there an issue with extended classes?

There should not be and I'm guessing we just did not look up the hierarchy for fields.

I will get this bug fixed today or tomorrow!

**Comment by agentgt** [comment]
@pabloperich I have a solution but it only allows `public` inherited fields.

This is the same for methods as well.

**Comment by agentgt** [comment]
@pabloperich the fix is in v1.3.6 which is in maven central (or will be soon).

**Comment by pabloperich** [comment]
Excellent, that's what I need. Thank you!
